src/main/java/org/commonjava/migrate/pathmap/MigrateOptions.java

```diff
@@ -359,6 +377,33 @@ private boolean validateOptions()
         return true;
     }
 
+    private static final SimpleDateFormat FILE_DATE_FORMAT = new SimpleDateFormat( "yyyyMMddHH" );
```

Comment:
Should use this SimpleDateFormat as local var as it is not thread-safe.

Reply:
Ok. UPdated.